Unix/tools/allocdump

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+#
+# Script to run to collect memory allocation information for all running OMI processes
+# Note that in order for this to work:
+#   1.  Omi processes must have been built with debug turned on
+#   2.  Omi loglevel must be INFO or higher
+#   3.  Current user needs to be enabled for sudo
+
+SCRIPTDIR=$(dirname "$0")
+BINDIR=/opt/omi/bin
+GDB=gdb
+SUDO=sudo
```

Comment:
Note above, this might be conditional based on current effective ID.

Reply:
Also note: User may have an alternate `sudo`. I know that `scxcore` has support for this, although I don't think we should check for that. You might want to just print a message up front that `gdb` would be run via `sudo`, if you'll really be doing that (i.e. not running as root user).